spec/models/pglogical_subscription_spec.rb

```diff
@@ -377,9 +377,7 @@ def with_an_invalid_remote_schema
       expect(remote_region.auth_key_configured?).to be true
 
       sub.delete
-      remote_region.reload
-
-      expect(remote_region.auth_key_configured?).to be false
+      expect(AuthToken.find_by_id(auth.id)).to be_nil
```

Comment:
Is this determining whether it can be retrieved rather than it actually existing?